gammapy/irf/energy_dispersion.py

```diff
@@ -924,3 +925,58 @@ def peek(self, figsize=(15, 5)):
         edisp.plot_matrix(ax=axes[2])
 
         plt.tight_layout()
+
+    def to_table(self, provenance=None):
+        """Offset-dependent energy dispersion matrix.
+
+        Data format specification: :ref:`gadf:edisp_2d`
+
+        Parameters
+        ----------
+        provenance : `list`
+            dict containing required information for fits header
+
+        Examples
+        --------
+        Read energy dispersion IRF from disk:
```

Comment:
There's formatting issues with your docstrings.

Locally I ran this:
```
make clean
python setup.py build_docs # very slow, 5 min
make doc-show
```
and see something like this:

![screen shot 2017-11-16 at 12 26 55](https://user-images.githubusercontent.com/852409/32889070-a86c4596-cac9-11e7-8582-54127de4006e.png)

Can you please fix up your docstrings?

I think in the examples section you have to do `::` to declare a code block and indent:
```
Here's my code::

    print('thank you for this pull request!')
```
or you can use `>>>` without indentation if it's just a few lines of code or you want to mix prose and code:
```
Here's some code:

>>> print('spam!')
```

Reply:
I did not know that would slow it down so much! I will clean it up.